last_seen.py

```diff
@@ -11,23 +11,39 @@ def seen_evt(self, event):
         events = ['quit', 'part', 'kick', 'join', 'rename', 'topic', 'message']
         if event.event_type in events:
             nick = event.meta['nick']
-            self.seen[nick] = (event.event_type, gmtime())
+            self.seen[nick] = (event.event_type, datetime.datetime.utcnow())
 
     @cmd
     def last_seen(self, event):
-        """
-        Returns when was given user last seen.
+        """Return when was given user last seen.
 
         Example: !last_seen <user>
         """
         args = event.args
         if len(args) < 1:
-            return "Name argument missing."
+            return 'Name argument missing.'
```

Comment:
@Adman This is not your fault but can you please replace this with something a bit more clear, like `'Username/nick not specified'`? 

The other possibility might be a serious usage message (`usage: !last_seen <nick>`). Feel free to hardcode `!` there for now but keep in mind that it is configurable and can be retrieved from `event.bot.command_token`.